tokenizers/src/pre_tokenizers/metaspace.rs

```diff
@@ -55,6 +120,9 @@ impl Metaspace {
         self.replacement = replacement;
         self.str_rep = replacement.to_string();
     }
+    pub fn set_prepend_scheme(&mut self, scheme: impl Into<PrependScheme>) {
+        self.prepend_scheme = scheme.into();
```

Comment:
Just make that `PrependScheme`.

`impl Into<T>` has an associated cost with it. Which is that there will be a concrete function for EVERY possible caller that might arise (here at least `PrependScheme`, `&str` and `String`. 

This is nice to make an API easier to use when the concrete underlying type is relatively verbose to make, or there are many structs implementing the given trait that might be useful (something like `impl Response` in a web framework.

Here everything is a simple `enum`. Importing the enum and using a variant makes the function  so that only 1 exists, prevents all kind of runtiem errors (all will become compile time errors). And since only 1 function exists, it can be inline more easily by the compiler.

Simple is better here IMO